packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js

```diff
@@ -185,6 +185,11 @@ MeetingInfoUtil.generateOptions = async (from) => {
     return MeetingInfoUtil.getSipUriFromHydraPersonId(hydraId.destination, webex).then((res) => {
       options.destination = res;
 
+      // Since hydra person ids require a unique case in which they are
+      // entirely converted to a SIP URI, we need to set a flag for detecting
+      // this type of destination.
+      options.wasHydraPerson = true;
```

Comment:
This flag allows us to detect if the destination is in fact a HydraId, since the output of this method, otherwise, provides information that is too generalized.

Reply:
Not sure if there are any tests around this method, but it would be nice to setup a unit test that verifies this property is set.